code/modules/mechs/equipment/utility.dm

```diff
@@ -350,9 +350,6 @@
 	to_chat(user, "It [get_visible_durability()].")
 
 
-/obj/item/drill_head/steel
```

Comment:
Because items are tested whether they're properly setting their health. Things without materials do not set their health, unless you specify it manually? Its not random.

Reply:
done for the drill